compiler/AST/iterator.cpp

```diff
@@ -989,6 +1019,10 @@ buildZip2(IteratorInfo* ii, Vec<BaseAST*>& asts, BlockStmt* singleLoop) {
       zip2body->insertAtTail(expr->copy(&map));
   }
 
+  // Since this passes isSingleLoopIterator(), there is a single yield,
+  // so 'more' is always the first value after initialization, i.e. 2.
```

Comment:
I thought 'more' was basically a boolean? What is it really?

Reply:
`IC.more` is a number indicating where the next `IC.advance()` should resume. The jumptable at the start of `IC.advance()` keys off of `IC.more`.

Reply:
One day we should rename it, then